osu.Game/Skinning/SkinnableSound.cs

```diff
@@ -28,6 +28,16 @@ public class SkinnableSound : SkinReloadableDrawable
         public override bool RemoveWhenNotAlive => false;
         public override bool RemoveCompletedTransforms => false;
 
+        /// <summary>
+        /// Whether to play the underlying sample when aggregate volume is zero.
+        /// Note that this is checked at the point of calling <see cref="Play"/>; changing the volume post-play will not begin playback.
+        /// Defaults to false unless <see cref="Looping"/>.
+        /// </summary>
+        /// <remarks>
+        /// Can serve as an optimisation if it is known ahead-of-time that this behaviour will not negatively affect behaviour.
+        /// </remarks>
+        protected bool SkipPlayWhenZeroVolume => !Looping;
```

Comment:
Not sure if I like this being a *negation* in nature (by which I mean this flag is `true` if samples with 0 volume should *not* be played). Kinda think it makes things harder to read (esp. the `!SkipPlayWhenZeroVolume` check - gotta flip it twice in my head to determine what will happen); I'd probably flip semantics and make it a `PlayWhenZeroVolume`.